usr.bin/sockstat/sockstat.c

```diff
@@ -1138,10 +1153,17 @@ format_unix_faddr(struct addr *faddr, char *buf, size_t bufsize) {
 				pos += snprintf(SAFEBUF, SAFESIZE,
```

Comment:
All of these snprintf calls aren't actually useful in json mode.  Can you conditionalize them?